Julia fails to build on on aarch64 because libquadmath.so is not available

Julia fails to build on aarch64 because `libquadmath.so` is not available on this arch.
Log:
```
[ 4706s] ln -sf /lib64/libgfortran.so.5 /home/abuild/rpmbuild/BUILD/julia-1.6.1/usr/lib/julia/libgfortran.so.5
[ 4706s] System library symlink failure: Unable to locate libquadmath.so.0 on your system!
[ 4706s] make[1]: *** [Makefile:206: /home/abuild/rpmbuild/BUILD/julia-1.6.1/usr/lib/julia/libquadmath.so.0] Error 1
[ 4706s] make[1]: Leaving directory '/home/abuild/rpmbuild/BUILD/julia-1.6.1/base'
[ 4706s] make: *** [Makefile:63: julia-base] Error 2
```

This is still broken with 1.10.3:
```
[15962s] '/home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/bin/7z' -> '/usr/bin/7z'
[15962s] ln -sf /usr/lib64/libopenlibm.so.4 /home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/lib64/julia/libopenlibm.so
[15962s] ln -sf /usr/lib64/libgcc_s.so.1 /home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/lib64/julia/libgcc_s.so.1
[15962s] ln -sf /usr/lib64/libgfortran.so.5 /home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/lib64/julia/libgfortran.so.5
[15962s] System library symlink failure: Unable to locate libquadmath.so.0 on your system!
[15962s] make[1]: *** [Makefile:246: /home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/lib64/julia/libquadmath.so.0] Error 1
[15962s] make: *** [Makefile:82: julia-base] Error 2
[15962s] make: *** Waiting for unfinished jobs....
```
But I cannot re-open.

> This is still broken with 1.10.3:
> 
> ```
> [15962s] '/home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/bin/7z' -> '/usr/bin/7z'
> [15962s] ln -sf /usr/lib64/libopenlibm.so.4 /home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/lib64/julia/libopenlibm.so
> [15962s] ln -sf /usr/lib64/libgcc_s.so.1 /home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/lib64/julia/libgcc_s.so.1
> [15962s] ln -sf /usr/lib64/libgfortran.so.5 /home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/lib64/julia/libgfortran.so.5
> [15962s] System library symlink failure: Unable to locate libquadmath.so.0 on your system!
> [15962s] make[1]: *** [Makefile:246: /home/abuild/rpmbuild/BUILD/julia-1.10.3/build/usr/lib64/julia/libquadmath.so.0] Error 1
> [15962s] make: *** [Makefile:82: julia-base] Error 2
> [15962s] make: *** Waiting for unfinished jobs....
> ```
> 
> But I cannot re-open.

I think you should open a ticket to the openSUSE bugzilla. like i said, libquadmath is disabled in opensuse. check out the specfile.

--- a/base/Makefile
+++ b/base/Makefile
@@ -249,12 +249,12 @@ endif
 ifneq (,$(LIBGFORTRAN_VERSION))
 $(eval $(call symlink_system_library,CSL,libgfortran,$(LIBGFORTRAN_VERSION)))
 endif
-$(eval $(call symlink_system_library,CSL,libquadmath,0))
 $(eval $(call symlink_system_library,CSL,libstdc++,6))
-# We allow libssp, libatomic and libgomp to fail as they are not available on all systems
+# We allow libssp, libatomic, libgomp and libquadmath to fail as they are not available on all systems
 $(eval $(call symlink_system_library,CSL,libssp,0,ALLOW_FAILURE))
 $(eval $(call symlink_system_library,CSL,libatomic,1,ALLOW_FAILURE))
 $(eval $(call symlink_system_library,CSL,libgomp,1,ALLOW_FAILURE))
+$(eval $(call symlink_system_library,CSL,libquadmath,0,ALLOW_FAILURE))
 $(eval $(call symlink_system_library,PCRE,libpcre2-8))
 $(eval $(call symlink_system_library,DSFMT,libdSFMT))
 $(eval $(call symlink_system_library,LIBBLASTRAMPOLINE,libblastrampoline))

